results/cmd/api/main_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	pb "github.com/tektoncd/experimental/results/proto/proto"
+	fieldmask "google.golang.org/genproto/protobuf/field_mask"
```

Comment:
This library is unused, let's remove it. (Running `goimports` should fix this automatically!)

Reply:
It is actually used in TestUpdatedTaskRun to create a fieldmask

Reply:
Looking at the implementation of Update, it doesn't look like this is used. =\

This actually points out something important though! We should probably validate the result of the update somehow, either by using Get or reading the DB.

Reply:
oh, you mean in the UpdateTaskRun function itself. I will delete it along with its definition in proto file.

Reply:
Done

Reply:
I also added a further test to see if values really have changed.